src/pages/components/pagination/usage.mdx

```diff
@@ -54,55 +59,377 @@ control for navigating to the next or previous page.
+| [Pagination](#pagination)         | The pagination variant is typically connected at the bottom of the data table component to help paginate large amounts of data. |
+| [Pagination nav](#pagination-nav) | The pagination nav variant is mainly used in on-page situations to help paginate either a whole page or sections of a page.     |     |
+
+## Formatting
+
+### Anatomy of pagination
+
+<Row>
+<Column colLg={12}>
+
+![Anatomy of the pagination variant](images/pagination-usage-anatomy.png)
+
+</Column>
+</Row>
+
+<Caption>Anatomy of the pagination variant</Caption>
+
+1. **Items per page:** Shows the current number of items that appear per page.
+2. **Range of items:** Shows the current range of items and lists the total
+   amount of items.
+3. **Current page:** Shows the current page the user is on and lists the total
+   amount of pages. 4.**Previous and next buttons:** Buttons that take you to
+   the previous or next page.
+4. **Container:** The container bar of the whole pagination component.
+
+### Anatomy of pagination nav
+
+<Row>
+<Column colLg={12}>
+
+![Anatomy of the pagination nav variant](images/pagination-usage-anatomy-nav.png)
+
+</Column>
+</Row>
+
+<Caption>Anatomy of the pagination nav variant</Caption>
+
+1. **Selected page button:** The current page the user is viewing.
+2. **Unselected page button:** A page the user could select and view.
+3. **Overflow button:** Contains available pages that can be navigated to but
+   cannot be shown upfront in the component because of the current viewport.
+4. **Previous and next buttons:** Buttons that take you to the previous or next
+   page.
+5. **Container:** The container of the whole pagination component.
+
+### Sizing
+
+The pagination component is available in three sizes—**large**, **medium**, and
+**small**.
+
+<Row>
+<Column colLg={12}>
+
+![Sizes of the pagination component](images/pagination-usage-sizes.png)
+
+</Column>
+</Row>
+
+<Caption>Sizes of the pagination component</Caption>
+
+We recommend using the same height pagination as you do for the data table
+component rows to which it is connected. In some cases, the height sizes of
+these two components will not be the same because we do not offer extra small
+and extra large sizes for pagination. When using the extra large data table row
+height, use the large pagination. When using the extra small data table row
+height, use the small pagination.
+
+<Row>
+<Column colLg={12}>
+
+![Size pairings of the pagination and data table components.](images/pagination-usage-size-pairings.png)
+
+</Column>
+</Row>
+
+<Caption>Size pairings of the pagination and data table components</Caption>
+
+### Placement
+
+Place the pagination near the related component or area of the page where the
+information will be paginated.
+
+When using the pagination variant with a data table, the pagination should be
```

Comment:
I feel like this paragraph and the one below could also use an h4 to help with scannability. Maybe this is something to figure out wtih the multi-variant content templates.
- H4: Placing pagination with data table
- H4: Placing pagination-nav on a page

Reply:
Also sometimes people ask about placing data table pagination at the top of the table. Would it be worth having a do/dont for that?

Reply:
Yes, that would be a good one!